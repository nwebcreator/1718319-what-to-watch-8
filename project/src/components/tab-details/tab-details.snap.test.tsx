import { render } from '@testing-library/react';
import TabDetails from './tab-details';

describe('Component: TabDetails', () => {
  it('should render correctly', () => {
    const film = {
      id: 1,
      name: 'string',
      posterImage: 'string',
      previewImage: 'string',
      backgroundImage: 'string',
      backgroundColor: 'string',
      videoLink: 'string',
      previewVideoLink: 'string',
      description: 'string',
      rating: 10,
      scoresCount: 10,
      director: 'string',
      starring: [],
      runTime: 10,
      genre: 'string',
      released: 2010,
      isFavorite: false,
    };
    const { container } = render(<TabDetails film={film} />);
    expect(container).toMatchSnapshot();
  });
});
