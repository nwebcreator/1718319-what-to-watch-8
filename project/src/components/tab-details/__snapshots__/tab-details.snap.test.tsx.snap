// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: TabDetails should render correctly 1`] = `
<div>
  <div
    class="film-card__text film-card__row"
  >
    <div
      class="film-card__text-col"
    >
      <p
        class="film-card__details-item"
      >
        <strong
          class="film-card__details-name"
        >
          Director
        </strong>
        <span
          class="film-card__details-value"
        >
          string
        </span>
      </p>
      <p
        class="film-card__details-item"
      >
        <strong
          class="film-card__details-name"
        >
          Starring
        </strong>
        <span
          class="film-card__details-value"
        >
          <span>
            
             
          </span>
        </span>
      </p>
    </div>
    <div
      class="film-card__text-col"
    >
      <p
        class="film-card__details-item"
      >
        <strong
          class="film-card__details-name"
        >
          Run Time
        </strong>
        <span
          class="film-card__details-value"
        >
          10m
        </span>
      </p>
      <p
        class="film-card__details-item"
      >
        <strong
          class="film-card__details-name"
        >
          Genre
        </strong>
        <span
          class="film-card__details-value"
        >
          string
        </span>
      </p>
      <p
        class="film-card__details-item"
      >
        <strong
          class="film-card__details-name"
        >
          Released
        </strong>
        <span
          class="film-card__details-value"
        >
          2010
        </span>
      </p>
    </div>
  </div>
</div>
`;
